diff --git a/node_modules/@nivo/bump/dist/nivo-bump.es.js b/node_modules/@nivo/bump/dist/nivo-bump.es.js
index a053933..d39d086 100644
--- a/node_modules/@nivo/bump/dist/nivo-bump.es.js
+++ b/node_modules/@nivo/bump/dist/nivo-bump.es.js
@@ -375,6 +375,7 @@ var useBump = function useBump(_ref3) {
         point.style = getPointStyle(_objectSpread2(_objectSpread2({}, point), {}, {
           serie: serie
         }));
+        point.hidden = serie.hidden;
         pts.push(point);
       });
     });
@@ -461,7 +462,7 @@ var useSeriesLabels = function useSeriesLabels(_ref5) {
       }
       labels.push({
         id: serie.id,
-        label: label,
+        label: serie.hidden ? '' : label,
         x: point[0] + signedPadding,
         y: point[1],
         color: getColor(serie),
@@ -500,7 +501,7 @@ var Point = function Point(_ref) {
   var animatedProps = useSpring({
     x: x,
     y: y,
-    radius: size / 2,
+    radius: _ref.hidden ? 0 : size / 2,
     color: color,
     borderWidth: borderWidth,
     config: springConfig,
@@ -652,8 +653,8 @@ var Line = function Line(_ref) {
   var animatedPath = useAnimatedPath(linePath);
   var animatedProps = useSpring({
     color: serie.color,
-    opacity: serie.style.opacity,
-    lineWidth: serie.style.lineWidth,
+    opacity: serie.hidden ? 0 : serie.style.opacity,
+    lineWidth: serie.hidden ? 0 : serie.style.lineWidth,
     config: springConfig,
     immediate: !animate
   });
@@ -671,7 +672,7 @@ var Line = function Line(_ref) {
     fill: "none",
     stroke: "red",
     strokeOpacity: 0,
-    strokeWidth: yStep,
+    strokeWidth: serie.hidden ? 0 : yStep,
     d: linePath,
     strokeLinecap: "butt",
     onMouseEnter: handlers.onMouseEnter,
@@ -732,6 +733,7 @@ var Points = function Points(_ref) {
     return React.createElement(pointComponent, {
       key: point.id,
       data: point.data,
+      hidden: point.hidden,
       x: point.x,
       y: point.y,
       isActive: point.isActive,
diff --git a/node_modules/@nivo/bump/index.d.ts b/node_modules/@nivo/bump/index.d.ts
index c25b0e0..9d221e4 100644
--- a/node_modules/@nivo/bump/index.d.ts
+++ b/node_modules/@nivo/bump/index.d.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /*
  * This file is part of the nivo project.
  *
