diff --git a/node_modules/@types/react-virtualized/dist/es/Table.d.ts b/node_modules/@types/react-virtualized/dist/es/Table.d.ts
index 569a02b..aaede89 100644
--- a/node_modules/@types/react-virtualized/dist/es/Table.d.ts
+++ b/node_modules/@types/react-virtualized/dist/es/Table.d.ts
@@ -354,7 +354,7 @@ export const SortDirection: SortDirectionStatic;
 
 export type SortDirectionType = 'ASC' | 'DESC';
 
-export const SortIndicator: React.StatelessComponent<{
+export const SortIndicator: React.FunctionComponent<{
     sortDirection?: SortDirectionType;
 }>;
 
